src/plugins/kibana_usage_collection/server/collectors/application_usage/rollups.ts

```diff
@@ -96,7 +96,7 @@ export async function rollDailyData(logger: Logger, savedObjectsClient?: ISavedO
           })),
           { overwrite: true }
         );
-        await Promise.all(
+        await Promise.allSettled(
```

Comment:
As this will no longer be throwing, should we retrieve the result of `allSettled` to log the potential failures?

Reply:
Good point! I've pushed an update to throw if we find that any promise was rejected

Reply:
Since this happens very frequently when there's multiple nodes, and users should always just ignore this message when it happens we decided to lower the log level to debug.